### DISCUSSION TITLE
请问那个笔记的内容是保存在哪里的？后台的文章里边没有.

### DISCUSSION THREAD

**Discussion by xhwwstudio** [Tokinx/Wing#61] [问答]
我的担心是，如果我以后换主题或者怎么样，这个笔记的数据保存在哪里吖？

**Comment by Tokinx** [comment]
> 我的担心是，如果我以后换主题或者怎么样，这个笔记的数据保存在哪里吖？

笔记不是文章，如果更换主题数据是还在，但是其它主题可能不支持展示笔记数据。

**Reply by shmilyabyy** [reply]
请问怎么删除这个笔记

**Comment by Snitro** [comment]
笔记是一种特殊类型的文章，与文章使用同样的数据存储方式，但是大概率只能通过这个主题展示在前端

**Comment by hongjr03** [comment]
笔记和说说能不能合并呢
